wikimedia/wikipedia-ios
# [YiR] Donate slide (Part 1 - data)

**Phabricator:** https://phabricator.wikimedia.org/T378838

### Notes
This PR checks for our new `donateCount` config key and populates a donate slide in the YiR report. This is pre-work to determine if we should add the personalized or collective donate slide. It is patterned after the read count slide logic.

### Test Steps
This is not really testable until the UI layer is in, but if you want, you could submit a donation before enabling the YiR developer settings toggle. Background, foreground to trigger report creation. Then when tapping the profile button, add a breakpoint [here](https://github.com/wikimedia/wikipedia-ios/pull/5046/files#diff-72caf34090dc13194bbfc66eeb449b72948cef8bd240c2a17c71437894632f90R183) to confirm that the donate personalized slide display property == true.
